cleanlab/multiannotator.py

```diff
@@ -275,26 +300,439 @@ def get_label_quality_multiannotator(
+                ]
+            ),
+            weights=[model_weight, avg_annotator_weight],
+            axis=0,
+        )
+
     else:
-        return {
-            "label_quality": label_quality,
-        }
+        active_learning_scores_unlabeled = np.array([])
+
+    return active_learning_scores, active_learning_scores_unlabeled
+
+
+def get_active_learning_scores_ensemble(
+    labels_multiannotator: Union[pd.DataFrame, np.ndarray],
+    pred_probs: np.ndarray,
+    pred_probs_unlabeled: Optional[np.ndarray] = None,
+) -> Tuple[np.ndarray, np.ndarray]:
+    """Returns an active learning score for each example in the dataset. The score is in between 0 and 1; lower scores indicate less confidence on the examples
+    and that additional labels should be collected for those examples. For example:
+
+    * 1 - confident example, no additional labeling is required.
+    * 0 - unconfident example, it is suggested to collect more labels for this example.
+
+    This function allows for an ensemble of models to be trained and will use the aggregation of predictions from the multiple models to compute the active learning score.
+
+    Parameters
+    ----------
+    labels_multiannotator : pd.DataFrame of np.ndarray
+        2D pandas DataFrame or array of multiple given labels for each example with shape ``(N, M)``,
+        where N is the number of examples and M is the number of annotators.
+        ``labels_multiannotator[n][m]`` = label for n-th example given by m-th annotator.
+
+        For a dataset with K classes, each given label must be an integer in 0, 1, ..., K-1 or ``NaN`` if this annotator did not label a particular example.
+        If you have string or other differently formatted labels, you can convert them to the proper format using :py:func:`format_multiannotator_labels <cleanlab.internal.multiannotator_utils.format_multiannotator_labels>`.
+        If pd.DataFrame, column names should correspond to each annotator's ID.
+    pred_probs : np.ndarray
+        An array of shape ``(P, N, K)`` where P is the number of models, consisting of predicted class probabilities from the ensemble models.
+        Each set of predicted probabilities with shape ``(N, K)`` is in the same format expected by the :py:func:`get_label_quality_scores <cleanlab.rank.get_label_quality_scores>`.
+    pred_probs_unlabeled : np.ndarray, optional
+        An array of shape ``(N, K)`` of predicted class probabilities from a trained classifier model for examples that have no annotator labels.
+
+    Returns
+    -------
+    active_learning_scores : np.ndarray
+        Array of shape ``(N,)`` indicating the active learning confidence scores for each example.
+    active_learning_scores_unlabeled : np.ndarray
+        Array of shape ``(N,)`` indicating the active learning confidence scores for each unlabeled example.
+        Returns an empty array if no unlabeled data is provided.
+    """
```

Comment:
I think we are going with the name "active learning quality scores" right?
If so, make sure to stick with this name throughout everything